ql/src/java/org/apache/hadoop/hive/ql/exec/repl/incremental/IncrementalLoadTasksBuilder.java

```diff
@@ -347,7 +327,7 @@ private boolean shouldReplayEvent(FileStatus dir, DumpType dumpType, String dbNa
       if (needCommitTx) {
         if (updateMetaData.getPartitionsList().size() > 0) {
           updateReplIdTask = getMigrationCommitTxnTask(dbName, tableName,
-                  updateMetaData.getPartitionsList(), replState, isDatabaseLoad, barrierTask);
```

Comment:
same chnages required in txn handler class also

Reply:
Done